assessment.rb

```diff
@@ -2,11 +2,40 @@
 # Assessment answers to:
 
 # question 2:
-
+licensee = licensee.create( surname: 'Bishop', given_name: 'Caren', height: 63, weight: 132, dob: '1943-09-26', gender: 'f')
+p licensee
 # question 3:
+class CreateLicensees < ActiveRecord::Migration
+  def change
+    create_table :licensees do |t|
+      t.string :surname
+      t.string :given_name
+      t.string :gender
+      t.integer :height
+      t.integer :weight
+      t.date :dob
+      t.string :eye_color
+      t.string :hair_color
 
+      t.timestamps null: false
+    end
+  end
+end
```

Comment:
This is a migration; migrations change the structure of a database, not its contents. What we want here is a script for generating new instances of Licensee using the data found in `data/licensees.csv`